src/applications/vaos/tests/appointment-list/components/CommunityCareAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -563,3 +569,215 @@ describe('VAOS <CommunityCareAppointmentDetailsPage>', () => {
     expect(tokens.get('END')).includes('VCALENDAR');
   });
 });
+
+describe('VAOS <CommunityCareAppointmentDetailsPage> with VAOS service', () => {
+  beforeEach(() => {
+    mockFetch();
+    MockDate.set(getTimezoneTestDate());
+  });
+  afterEach(() => {
+    MockDate.reset();
+  });
+
+  it('should navigate to community care appointments detail page', async () => {
+    const url = '/cc/01aa456cc';
+    const appointmentTime = moment().add(1, 'days');
+    const start = moment()
+      .subtract(30, 'days')
+      .format('YYYY-MM-DD');
+    const end = moment()
+      .add(395, 'days')
+      .format('YYYY-MM-DD');
+
+    const appointment = getVAOSAppointmentMock();
```

Comment:
Not at all required, but you could try out the new createAppointmentMockByVersion function I created, since that's merged now.